src/scripts/party/PartyPokemon.ts

```diff
@@ -359,6 +359,10 @@ class PartyPokemon implements Saveable {
         if (!new RegExp(Settings.getSetting('heldItemSearchFilter').observableValue() , 'i').test(this.displayName)) {
             return true;
         }
+        if (Settings.getSetting('heldItemHideHoldingPokemon').observableValue() && typeof(this.heldItem()) !== 'undefined') {
```

Comment:
`&& typeof(this.heldItem()) !== 'undefined'` can just be `&& this.heldItem()`

Reply:
ah thanks, i wasn't sure how undefined would work so my old JS ways recalled this existing x)